samples/nrf9160/mqtt_simple/src/main.c

```diff
@@ -266,12 +310,10 @@ static void broker_init(void)
 	err = getaddrinfo(CONFIG_MQTT_BROKER_HOSTNAME, NULL, &hints, &result);
 	if (err) {
 		printk("ERROR: getaddrinfo failed %d\n", err);
-
-		return;
+		return err;
```

Comment:
Note that `getaddrinfo` returns positive error codes

Reply:
Changed to -err;

Reply:
Okay, that's confusing. Better translate it to a negative error code than (as I see you've already done)